apps/service_discovery_http/src/sdh_conn.erl

```diff
@@ -0,0 +1,23 @@
+-module(sdh_conn).
+
+-behaviour(acceptor).
+
+-export([acceptor_init/3,
+         acceptor_continue/3,
+         acceptor_terminate/2]).
+
+acceptor_init(_, LSocket, {Transport, ElliCallback, ElliOpts, SslOpts}) ->
+    % monitor listen socket to gracefully close when it closes
+    MRef = monitor(port, LSocket),
```

Comment:
this isn't used anywhere?

Reply:
Oh, hm, may have used that in grpc for dealing with bidi streams to know to start shutting down, copied over much of this code from there.